### DISCUSSION TITLE
一个小建议

### DISCUSSION THREAD

**Discussion by Yukari316** [CollegesChat/university-information#79] [Enhancement]
QA其实可以更详细一些

## 宿舍相关
宿管的管理方式
电费是如何度量

## 娱乐相关
学校的校区是否偏远
学校的校区周围有多少商业区
和商业区的距离又有多远
去商业区方便吗？消费水平呢？

## 不同的校区
有一部分的学校的不同专业是在不同的校区的
`比如我所在的大学就是如此`
这个学校的住宿条件只有一个校区比较好其他校区都很一般

## 学校的网络
情况其实是比较难以用一个问题去衡量的（比如网速和网络开放时间）
学校的网络情况其实有很多方面可以去衡量
随便列举几个（

- 学校网络的ISP
- 学校网络的DNS污染/劫持情况
- 学校网络是否是由多个ISP组成一个单一的线路，或者说走单一网关（比如我的学校就是，在流量大的时候丢包直接上升到60%左右）
- 学校的网络是否限速
- 学校WiFi/4G/5G信号的覆盖情况
- 不同ISP提供的套餐费用和流量以及合同时长

这些其实乍一看都是一些很难以注意到的小问题，但是这都是我在学校网络中遇到的各种小问题
非常的烦人其实

## 学校的学院压力，学院水平
讲道理我自我觉得我自己的学校学院水平很不好（比如计算机相关）
我在大学期间的技术几乎都是我自学的，老师的技术水平完全停留在10年前
这类的学校对学生的自学能力要求非常高（简单来说就是没法从老师那学东西

我想说的建议就是这些，希望闲暇之余可以看看
![8E27548C3D22A5E535FA3CF3009CF149](https://user-images.githubusercontent.com/7535224/123555324-a3adde80-d7b7-11eb-85a4-784df20d59e5.jpg)

**Comment by ghost** [comment]
支持

**Comment by TerraceCN** [comment]
其实对于大部分学校而言，真正有用的东西都是自学得来的（

**Reply by The-Fae-Sorceress** [reply]
专业知识需要从老师那学，其他的大部分要自学或者从师兄师姐那里学一学

**Reply by Yukari316** [reply]
确实
